specificatie/brp-api/gezag/gezagsrelatie-v1.yaml

```
@@ -70,7 +70,7 @@ components:
             derden:
               type: array
               items:
-                $ref: 'meerderjarige-v1.yaml#/components/schemas/Meerderjarige'
+                $ref: 'derde-v1.yaml#/components/schemas/AbstractDerde'
```

Zoals overlegd tijdens de dag start zal voogdij een array van bekende derden bevatten. Bij onbekende derde(n) zal de array leeg zijn.

```suggestion
                $ref: 'derde-v1.yaml#/components/schemas/BekendeDerde'
```

---

Ik kan mij helemaal vinden in de voorgestelde wijzigingen (Derde/BekendeDerde/OnbekendeDerde als types en collectie van BekendeDerde bij Voogdij type). Dit zal volgens mij ook bijdragen aan json data die aansluit bij het domein.

Ik denk dat als er een breaking change komt dat we al de AbstractXXX types moeten hernoemen, zodat de technische term (Abstract) verdwijnt uit de namen van de schema definities.

@CathyDingemanse, @fsamwel, ik zat mij af te vragen of het hernoemen van TweehoofdigOuderlijkGezag naar GezamenlijkOuderlijkGezag ook niet al kan worden gedaan op de gezag-api en de personen-data-service. Die zijn (nog) interne services die maar één consumer hebben en deze wijziging zou een kleine impact hebben (in principe hoeft alleen voor de personen-informatie-service voor deze wijziging te worden aangepast). Bijkomend voordeel is dat als in de toekomst bij een breaking change de hernoeming moet worden doorgevoerd, dan hoeft dit ook alleen te gebeuren in de informatie service en niet meer bij de data service en gezag api